### ISSUE TITLE
how to use with algebra.js

### ISSUE THREAD

**Issue by husayt** [josdejong/mathjs#1068]
[Algebra.js](https://github.com/nicolewhite/algebra.js) allows solving many equation types very nicely and has a similar expression tree and parsing. Would be great to be able to integrate with math.js like we do with numbers.js.

How could this be done?

**Comment by josdejong** [comment]
Yes indeed, Algebra.js is a great initiative. So far it's simply two parallel paths. I don't know if/how we could join forces.

In #442 we shortly discussed together with the creator of Algebra.js but that did not result in anything so far.

If anyone is interested in picking this up to see if there are possibilities just let me know.

**Comment by javazen** [comment]
I did some experimentation with algebra.js in conjunction with math.js. I found a couple of limitations that prevented me from going further with it:
1) algebra doesn't seem to parse functions: new algebra.parse("2*abs(x)").toString() -> "2absx"
2) algebra doesn't parse expressions with a variable in the denominator, e.g. 1/x

I don't see how I can get around these limitations, as much as I would like to make use of the cool term-collecting and solving in algebra.js.
